src/util/util_string.h

```diff
@@ -5,6 +5,10 @@
 #include <string>
 #include <sstream>
 
+#ifdef __WINE__
+#include "windows.h"
```

Comment:
If anything needs windows.h, please don't include `windows.h` directly but use the `com_include.h` header which sets up a few defines beforehand.